readthedocs/ethical-ad-server
# Offload the advertiser geo report to metabase

- Still reports on a geo (not region) basis
- Works for basically any time frame even big ones on huge advertisers (no timeouts)
- I don't love the ranges metabase chooses but it doesn't look customizable. You can customize the colors but not set thresholds

## Testing

- Set `METABASE_SITE_URL=https://ethicalads.metabaseapp.com` and `METABASE_SECRET_KEY` in `.envs/local/django`
- Ensure an advertiser slug locally matches a production advertiser slug
- Load the geo report

## Screenshot
![Screenshot from 2022-07-29 15-39-28](https://user-images.githubusercontent.com/185043/181854382-6a079a74-36f9-4513-afe2-920a3c8b092e.png)
